pucoti.py

```diff
@@ -647,12 +654,19 @@ def save_last_purpose():
                     else:
                         scene = Scene.MAIN
                 elif event.key == pg.K_j:
-                    timer -= 60
+                    timer -= 60 * 5 if shift_is_pressed() else 60
                 elif event.key == pg.K_k:
-                    timer += 60
+                    timer += 60 * 5 if shift_is_pressed() else 60
+                elif event.key in NUMBER_KEYS:
```

Comment:
I would also set `initial_duration` to the new duration here, so that reset uses the new duration. It seems a bit better